extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -128,7 +133,84 @@ public RepositoryAdapter build() {
       return new RepositoryAdapter(this);
     }
 
+    /**
+     * Builds the {@link RepositoryAdapter} that presents the provided repositories in order and
+     * observes the repositories as well as any additional observables while the provided
+     * {@link Activity} is active (between onResumed/onPaused).
+     */
+    @TargetApi(14)
+    @NonNull
+    public Adapter<ViewHolder> whileActive(@NonNull final Activity activity) {
```

Comment:
I speak source code ;)

https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/app/Activity.java#716

(interestingly, both the mResumed field and the `<em>active</em>` javadoc were checked in at the same time on 2009-03-03, so there seems to be some internal term disparity already)

I think I'm still in favor of 'resumed' to stay away from 'active', a term we use for Observables that are more general than the activity's 'active' state.